halconfig/clouddriver-caching.yml

```diff
@@ -0,0 +1,15 @@
+# halconfig
+
+admin.tasks.shutdownWaitSeconds: 600 # 10 minutes
```

Comment:
Is there a reason not to rely on the base clouddriver.yml for these shared properties?